packages/expo-sqlite/src/__tests__/SQLiteDatabase-test.ios.ts

```diff
@@ -42,7 +42,14 @@ describe('Database', () => {
 
   it('execAsync should throw error from an invalid command', async () => {
     db = await openDatabaseAsync(':memory:');
-    await expect(db.execAsync('INVALID COMMAMD')).rejects.toThrow();
+    let error: any;
+    try {
+      await db.execAsync('INVALID COMMAMD');
+    } catch (e: any) {
+      error = e;
+    }
+    expect(error).toBeDefined();
+    expect(error.toString()).toContain('syntax error');
```

Comment:
not quite sure why it starts to happen. the problem was that better-sqlite3's `SQLiteError` is not an instanceof Error. maybe because of prebuilt better-sqlite3 or incosistent nodejs version.

this manual try-catch does the trick